src/engine/SCons/Tool/packaging/__init__.py

```diff
@@ -163,15 +163,21 @@ def load_packager(type):
         # this exception means that a needed argument for the packager is
         # missing. As our packagers get their "tags" as named function
         # arguments we need to find out which one is missing.
-        from inspect import getargspec
-        args,varargs,varkw,defaults=getargspec(packager.package)
-        if defaults!=None:
-            args=args[:-len(defaults)] # throw away arguments with default values
+        #TODO: getargspec deprecated in Py3. cleanup when Py2.7 dropped.
+        try:
+            from inspect import getfullargspec
```

Comment:
Won't this fail if getfullargspec is actually imported instead of getargspec?

Reply:
yes... the thought was there but didn't express fully in the patch.  it needs an "as getargspec" or some other similar for the actual reference to resolve.

Reply:
I'll look to see if there's a test that actually tickles this, because testing with py3 is not failing this.